src/ResourceManager/Network/Commands.Network.Test/ScenarioTests/ApplicationGatewayTests.cs

```diff
@@ -30,6 +30,13 @@ public ApplicationGatewayTests(ITestOutputHelper output)
 
         [Fact]
         [Trait(Category.AcceptanceType, Category.CheckIn)]
+        public void TestAvailableWafRuleSets()
+        {
+            NetworkResourcesController.NewInstance.RunPsTest(string.Format("Test-AvailableWafRuleSets -baseDir '{0}'", AppDomain.CurrentDomain.BaseDirectory));
```

Comment:
yes, but it is not uploaded yet, I may change the cmdlet before it gets finalized.